pyproject.toml

```diff
@@ -0,0 +1,72 @@
+[project]
+name = "opengeh-utilities"
```

Comment:
It's gonna contain more than utilities. So perhaps `opengeh-common`? That also aligns somehow with the term `testcommon` for stuff that is common regarding testing.